site/pages/gc-subway-index.hbs

```diff
@@ -0,0 +1,100 @@
+---
+{
+"title": "GC-Subway Index",
+"language": "en",
+"altLangPrefix": "gc-subway-index",
+"breadcrumb": [
+{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" },
+{ "title": "Business and industry", "link": "https://www.canada.ca/en/services/business.html" },
+],
+"tag": "GCWeb",
+"dateModified": "2020-11-16",
+"share": "true"
+}
+---
+<h1 property="name" id="wb-cont" class="provisional gc-thickline">GC-Subway Index
+</h1>
+<div class="row">
+	<div class="col-md-8">
+		<div class="alert alert-info">
+			<p>This is an optional Alert</p>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+				ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+				ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+				reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
+				sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+				est laborum.</p>
+		</div>
+		<div class="panel panel-default brdr-rds-0">
+			<div class="panel-body">
+				<div class="row">
+					<div class="col-sm-5 col-lg-6">
+						<p class="mrgn-tp-sm">Text for Call-to-action button
+							Lorem ipsum dolor sit amet</p>
+					</div>
+					<div class="col-sm-7 col-lg-6">
+						<p><a href="#" class="btn btn-call-to-action">Call to action button</a></p>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<nav class="provisional gc-subway">
```

Comment:
Could be a div or a section here:
```suggestion
		<section class="provisional gc-subway">
```

Reply:
@GormFrank What is the rational of generalizing it? 

That section is primarily intended to be a major navigation block inside the main page. It is also the main task for that page.

> The nav element represents a section of a page that links to other pages or to parts within the page: a section with navigation links.

ref. https://html.spec.whatwg.org/multipage/sections.html#the-nav-element

Reply:
@duboisp Whatever works, just tought definition list (`<dl>`) in navigation element (`<nav>`) was pretty uncommon and sounds a bit off.